src/js/controller/model.js

```diff
@@ -383,8 +388,8 @@ define([
 
         };
 
-        this.persistVideoSubtitleTrack = function(trackIndex) {
-            this.setVideoSubtitleTrack(trackIndex);
+        this.persistVideoSubtitleTrack = function(trackIndex, tracks) {
+            this.setVideoSubtitleTrack(trackIndex, tracks);
```

Comment:
@robwalch @hghazzi yeah, we do. This behavior hasn't changed. When the cc menu selection changes, `setVideoSubtitleTrack` tells the provider what track should be selected. In the case of `TextTrack.change` events where the provider notifies the model, the provider returns early when `_provider.setSubtitlesTrack(trackIndex);` gets called in `setVideoSubtitleTrack `.